### ISSUE TITLE
Sonic Screwdriver doesn't rotate sign text when using signs_lib

### ISSUE THREAD

**Issue by dontfwiththestream** [mt-mods/technic#330]
Expected behavior

Text on sign rotates when you use sonic screwdriver on sign

Actual behavior

Sign rotates, leaving text position unchanged

Steps to reproduce:

Start server with signs_lib and technic_plus
write text on sign
rotate with sonic screwdriver

**Comment by OgelGames** [comment]
Yeah, this is a side effect of the sonic screwdriver's intentionally bad design. The sonic screwdriver doesn't call the `on_rotate` callback, for the same reason it doesn't call `can_dig`. See also https://github.com/mt-mods/technic/issues/267.
https://github.com/mt-mods/technic/blob/f5949f6bf40a0a945bf0c76fd9ba169e0bd0b2d0/technic/tools/sonic_screwdriver.lua#L39-L40

It's better just to use the [`omnidriver`](https://content.minetest.net/packages/OgelGames/omnidriver/) ;)
